### PR TITLE
Added doc for smooth follow

### PR DESCRIPTION
**Pull request by fjolidon**
Added documentation of the new followSmoothness field of the viewpoint node.

### CODE REVIEW

**Comment by fjolidon** [comment]
:+1:

File: reference/viewpoint.md [hunk 1]
```diff
@@ -53,5 +54,7 @@ adding the instance number to the `follow` field value using the format
 
 The `followOrientation` field can be used to make the viewpoint follow also the orientation of an object (in addition to its position). If `followOrientation` is true, the viewpoint is rigidly attached to the followed object, like an embedded camera onboard a robot. The `follow` field should be set with a valid object name otherwise the `followOrientation` field has no effect.
 
+The `followSmoothness` defines how smooth the camera is when following an object. With a value of 0 the camera follows the movement of the object instantly, and increasing the value increases the intertia of the camera thus making the movement more fluid. Higher values can sometimes make the camera lag behind in the case of fast moving objects that change direction often. This parameter is ignored if the `followOrientation` field is set to `TRUE`.
```
Comment (stefaniapedrazzi) [inline reference/viewpoint.md]: Could you please write each sentence on a new line?

Other comments:
* The 'followSmoothness' **field** defines...
* intertia -> inertia

File: guide/the-user-interface.md [hunk 2]
```diff
@@ -131,7 +131,10 @@ The `Follow Object` menu item allows you to switch between a fixed (static)
 viewpoint and a viewpoint that follows a mobile object (usually a robot). If you
 want the viewpoint to follow an object, first you need to select the object with
 the mouse and then check the `Follow Object` menu item. Note that the `Follow
-Object` state is saved in the ".wbt" file.
+Object` state is saved in the ".wbt" file. By default, the viewpoint will follow
+the object smoothly, mitigating shaking or jerky movements of the object. You can
+change this behavior by changing the `followSmoothness` field of the Viewpoint in the
```
Comment (stefaniapedrazzi) [inline guide/the-user-interface.md]: I would add the word `node` -> "... field of the Viewpoint **node** in the scene tree."

**Comment by fjolidon** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
This PR is fine, but it should only be merged after https://github.com/omichel/webots/pull/5319 has been accepted.

**Comment by fjolidon** [comment]
Oh ok. I thought I had to do this one first because in that PR you mentioned the doc should be updated before merging the PR.

**Comment by stefaniapedrazzi** [comment]
Yes, it is tricky because we want to merge both at the same time.
This PR has to merged before omichel/webots#5319, but only after the new feature is approved and ready to be merged.
https://github.com/omichel/webots/wiki/Git-workflow#submodules

**Comment by fjolidon** [comment]
The corresponding PR on the webots repo has already been merged... Should I create a new PR on the webots repo to update the doc submodule with the content of this PR?

**Comment by stefaniapedrazzi** [comment]
Yes, thank you.

**Pull request merged**